infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/SessionManager.java

```diff
@@ -140,12 +141,14 @@ public synchronized void refresh(@Nonnull TopologyDescriptor newTopology) {
         //  hashcode in all TopologyDescriptor and all its members.  It is being taken care of in a subsequent PR
         //  which contains changes to the Connection Model.
 
-        Set<String> newSources = newTopology.getSourceClusters().keySet();
-        Set<String> currentSources = topology.getSourceClusters().keySet();
+        // TODO: Shama - have logic to add a remote cluster -> part of connectionModel PR
```

Comment:
can you elaborate what this TODO means?

Reply:
this is not needed any more..its covered in the sessionManager PR. Removed it.